pkg/archive/archive.go

```diff
@@ -0,0 +1,184 @@
+package archive
```

Comment:
With regard to portability, I think that this might best added as a subpackage to bundle. Thoughts?

Reply:
@afflom I considered it a functional component outside of the bundle that interacts with bundles, but I would be willing to move it under bundle as sub package.

Reply:
I vote making it a sub-package. I asked Eric for his opinion.

Reply:
Since we don't yet know how these packages/command will look when integrating with `oc`, this is fine for now. Once we know those requirements, we can reorganize packages if needed.